flanker/addresslib/plugins/hotmail.py

```diff
@@ -82,10 +86,6 @@ def validate(email_addr):
     if localpart.count('+') > 1:
         return False
 
-    # no consecutive periods (..)
```

Comment:
This check is not required because consecutive periods are not allowed in dot-atoms anyway.